app/modules/ph/models_ph.py

```diff
@@ -1,14 +1,13 @@
 from datetime import date
 
-from sqlalchemy import Date, String
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped
 
-from app.types.sqlalchemy import Base
+from app.types.sqlalchemy import Base, PrimaryKey
 
 
 class Paper(Base):
     __tablename__ = "ph_papers"
 
-    id: Mapped[str] = mapped_column(String, primary_key=True, index=True)
-    name: Mapped[str] = mapped_column(String, nullable=False)
-    release_date: Mapped[date] = mapped_column(Date, nullable=False)
+    id: Mapped[PrimaryKey]
+    name: Mapped[str]
```

Comment:
Is it nullable or did the default value changed ?

Reply:
SQL column is automatically not nullable if the python type is not nullable.